openpype/hosts/hiero/api/lib.py

```diff
@@ -29,12 +28,22 @@
 from openpype.pipeline.load import filter_containers
 from openpype.lib import Logger
 from . import tags
-
+from .constants import (
+    OPENPYPE_TAG_NAME,
+    DEFAULT_SEQUENCE_NAME,
+    DEFAULT_BIN_NAME
+)
 from openpype.pipeline.colorspace import (
     get_imageio_config
 )
 
 
+class CTX:
+    _has_been_setup = False
+    _has_menu = False
+    _parent_gui = None
+
```

Comment:
Rather change the class to private (so it's clear it should not be imported) and remove privacy or attributes.
```suggestion
class _CTX:
    has_been_setup = False
    has_menu = False
    parent_gui = None

```